crates/sncast/tests/e2e/deploy.rs

```diff
@@ -344,3 +344,138 @@ fn test_too_low_max_fee() {
         "},
     );
 }
+
+#[test]
+fn test_no_class_hash_and_no_name() {
+    let tempdir = copy_directory_to_tempdir(CONTRACTS_DIR.to_string() + "/map");
+    let accounts_file = get_accounts_path(ACCOUNT_FILE_PATH);
+
+    let args = vec![
+        "--accounts-file",
+        accounts_file.as_str(),
+        "--url",
+        URL,
+        "--account",
+        "user1",
+        "deploy",
+    ];
+
+    let snapbox = runner(&args).current_dir(tempdir.path());
+    let output = snapbox.assert().failure();
+
+    assert_stderr_contains(output, "Error: Contract name unspecified");
```

Comment:
Shouldn't the error be something along `"Contract name and hash unspecified"`? This one suggests that name is missing, where it's just one of the options.

Reply:
Indeed - the absence of contract-name in this context implies the absence of class-hash too :)